pull-requests.md

```diff
@@ -22,6 +16,26 @@ The content changes go through the following stages before they are published on
 2. **Review:** Your changes are reviewed by [MDN members and volunteers](#pull-request-review-process).
 3. **Publishing:** Updated content goes live within a day of merging via a site rebuild once every 24 hours.
 
+## Before you start
+
+- **Pull requests must resolve or partially fix an existing issue.**
```

Comment:
```suggestion
## Before you start

Before you start work on MDN, please go through the recommendations and guidelines listed below.

**Pull requests must resolve or partially fix an existing issue.**
```
- Added a lead-in sentence between the two titles. Also because otherwise the paragraph begins with warning-sounding text in bold 🙂. 
- Un-bulleted the first point because it is not conveying the same type of info as the other three.

Reply:
Better, tnx